src/components/Menu/LanguageDropdownSelector/LanguageDropdownSelector.tsx

```diff
@@ -32,9 +33,11 @@ export const LanguageDropdownSelector = ({
   const isSelectedLanguage = (option: ReactSelectOption) => option.value === language;
   const isNotSelectedLanguage = (option: ReactSelectOption) => option.value !== language;
   let options = languages.map(makeOptionFromLang);
+  options = options.filter(({ value }) => isInImageMap(value));
```

Comment:
I love the creativity of using this but I don't think this is quite right, I think languages without images should be OK;

Can you add:
    const CURL = 'curl';
    const SHELL_LANGUAGES = [SH_LANGUAGE, BASH_LANGUAGE, ZSH_LANGUAGE, CURL];
in `data/createPages/constants.ts`

And then import `IGNORED_LANGUAGES_FOR_DISPLAY` and filter against that?

Reply:
@MymmiJ  amazing thanks for this, didnt know we have this, this is helpful. thanks Jimmy and updated the code